src/Storage.cpp

```diff
@@ -47,7 +47,7 @@ typedef struct __attribute__((packed)) {
     double brewPIDDelaySec; 
     uint8_t freeToUse10;
     double brewDetectionThreshold;
-    double wifisetup;
+    uint8_t WifiCredentialsSaved;
```

Comment:
good but needs to start lowercase, capital letters are usually reserved for class names ;)